baselines/aad.md

```diff
@@ -17,15 +19,14 @@ hosts the M365 resources being used.
 [home tenant](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/authentication-conditional-access)
 is the one that owns the external user’s (e.g., guest) account.
 
+## Highly Privilged Roles
+
+The following built-in Azure AD roles are considered highly privileged at a minimum: Global Administrator, Privileged Role Administrator, User Administrator, SharePoint Administrator, Exchange Administrator, Hybrid Identity Administrator, Application Administrator, Cloud Application Administrator. Additional built-in roles that are considered highly privileged in the agency's environment can be added to this list.
```

Comment:
I am not sure what "can be added to this list" means.  Do we have a way to add to privileged users and effect policy evaluation?

Reply:
I modified the paragraph for the Highly Privileged Roles section. See if the new content is clearer.